vespa-http-client/src/main/java/com/yahoo/vespa/http/client/core/communication/ApacheGatewayConnection.java

```diff
@@ -406,7 +406,7 @@ public HttpClient createClient() {
             }
             clientBuilder.setMaxConnPerRoute(1);
             clientBuilder.setMaxConnTotal(1);
-            clientBuilder.setConnectionTimeToLive(15, TimeUnit.SECONDS);
+            clientBuilder.setConnectionTimeToLive(10, TimeUnit.MINUTES);
```

Comment:
40x sounds like a big change. Will it impact error detection/load balancing ?
I would probably start with 1 minute as a 4x change.

Reply:
Ok, let's try with one minute first. I'll give you the motivation for this change offline.